lib/linter-integration.js

```diff
@@ -31,13 +31,15 @@ class Linter {
     }
     this.linter.setMessages(messages.map(match => ({
       type: match.type || 'Error',
-      text: match.message || 'Error from build',
+      text: !match.message && !match.html_message ? 'Error from build' : match.message,
+      html: match.html_message,
```

Comment:
And, of course, there's a third option - just ignore either `message` or `html_message` when both are set. It will reduce quantity of errors seen by the end user, but it will also hide problems in the provider code. Maybe it's better to show an error and let the provider developer know that he or she did something wrong.

Another alternative would be to ignore `message` if it equals to `html_message` and only use the latter. I think `html_message` must be preferred because in case of html-tags absence it behaves the same way as `message`.

What do you think?